extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -33,6 +34,12 @@ const TAG = 'amp-sidebar toolbar';
 /** @private @const {number} */
 const ANIMATION_TIMEOUT = 350;
 
+/** @private @const {string} */
+const LEFT = 'left';
```

Comment:
nit: I would prefer something more like:

```javascript
/** @private @enum {string} */
const Side = {
  LEFT: 'left',
  RIGHT: 'right',
};
```

This also allows you to change the type of the param for `setSideAttribute_` to `{!Side}`, instead of any arbitrary string.